src/asm/parser.y

```diff
@@ -810,13 +809,11 @@ align		: T_OP_ALIGN uconst {
 
 opt		: T_POP_OPT {
 			lexer_SetMode(LEXER_RAW);
-		} opt_list {
-			lexer_SetMode(LEXER_NORMAL);
-		}
+		} opt_list
```

Comment:
I think a comment noting that the lexer will switch back to normal mode on its own at EOL. Also, this means that `opt` will have the newline injection hack? Hm.

Reply:
`opt` having the newline injection hack was already the case. I'll add comments to it and `macroargs`.